modules/apps/object/object-web/src/main/resources/META-INF/resources/list_type_definitions/configuration/icon/import_list_type_definition.jsp

```diff
@@ -8,22 +8,13 @@
 <%@ include file="/init.jsp" %>
 
 <div>
-
-	<%
-	Map<String, Object> warningModalText = HashMapBuilder.<String, Object>put(
-		"body", Arrays.asList(LanguageUtil.format(request, "there-is-another-x-with-the-same-external-reference-code-as-the-imported-one", StringUtil.toLowerCase(LanguageUtil.get(request, "picklist"))), LanguageUtil.format(request, "before-importing-the-new-x-you-may-want-to-back-up-its-entries-to-prevent-data-loss", StringUtil.toLowerCase(LanguageUtil.get(request, "picklist"))), LanguageUtil.get(request, "do-you-want-to-proceed-with-the-import-process"))
-	).put(
-		"header", LanguageUtil.get(request, "update-existing-picklist")
-	).build();
-	%>
-
 	<react:component
 		module="js/components/ModalImport"
 		props='<%=
 			HashMapBuilder.<String, Object>put(
 				"apiURL", "/o/headless-admin-list-type/v1.0/list-type-definitions/by-external-reference-code/"
 			).put(
-				"externalReferenceCodeFeedbackMessage", LanguageUtil.get(request, "unique-key-for-referencing-the-picklist-definition")
+				"importedEntity", LanguageUtil.get(request, "picklist")
```

Comment:
I think we could rename this property because `LanguageUtil.get(request, "picklist")` represents a Label/Translated Name.
What do you think about `label` ? Because I saw that there is a property `title`, and this property changes depending on the entity/dto.

Reply:
Hey @marcelabc, I completely agree. Thanks!